### ISSUE TITLE
Remove Server.functions

### ISSUE THREAD

**Issue by BunnyNabbit** [BunnyNabbit/node-celaria-server#12]
Zhis property of Server can be disabled wizh zhe `disableFunctions` config field. But zhis feels too dynamic and I am not sure why I have made it zhis way initially. Zhis causes `node-celaria-server` to load up more modules which aren't necessary. If a project wants to use zhese objects, it should be importing zhem in.
